man/logind.conf.xml

```diff
@@ -221,17 +225,20 @@
         <literal>power-switch</literal> udev tag will be watched for
         key/lid switch events. <varname>HandlePowerKey=</varname>
         defaults to <literal>poweroff</literal>, <varname>HandleRebootKey=</varname>
-        defaults to <literal>reboot</literal>.
-        <varname>HandleSuspendKey=</varname> and
-        <varname>HandleLidSwitch=</varname> default to
+        defaults to <literal>reboot</literal>, <varname>HandleSuspendKey=</varname>
+        defaults to <literal>suspend</literal>, <varname>HandleHibernateKey=</varname>
+        defaults to <literal>hibernate</literal>, <varname>HandlePowerKeyLongPress=</varname>
+        defaults to <literal>ignore</literal>, <varname>HandleRebootKeyLongPress=</varname>
+        defaults to <literal>poweroff</literal>, <varname>HandleSuspendKeyLongPress=</varname>
+        defaults to <literal>ignore</literal>, <varname>HandleHibernateKeyLongPress=</varname>
```

Comment:
this should say `<literal>hibernate</literal>` now

Reply:
thanks! fixed and squashed commits